lib/Parser/Parse.cpp

```diff
@@ -2252,7 +2301,26 @@ void Parser::AddModuleImportEntry(ModuleImportEntryList* importEntryList, IdentP
     importEntryList->Prepend(*importEntry);
 }
 
-void Parser::AddModuleExportEntry(ModuleExportEntryList* exportEntryList, IdentPtr importName, IdentPtr localName, IdentPtr exportName, IdentPtr moduleRequest)
+void Parser::CheckForDuplicateExportEntry(ModuleExportEntryList* exportEntryList, IdentPtr exportName)
+{
+    Js::PropertyId newPropertyId = EnsurePropertyId(exportName);
```

Comment:
Okay. Actually, we should talk about this more in person, so I understand the cases we have to handle. I don't want to hold up the change for this if it's functionally correct, but can we open an issue to get rid of the n^2 search?